TAF_EasyRest/TestFramework/Pages/Client/CurrentOrdersPage.cs

```diff
@@ -6,30 +6,35 @@ namespace TestFramework.Pages
     public class CurrentOrdersPage : BasePage
     {
         public NavigationMenuPageComponent NavigationMenuPageComponent { get; }
+        public UserMenuHeaderButtonPageComponent UserButton { get; }
         public PersonalInfoPageComponent PersonalInfoPageComponent { get; }
         public List<WaitingForConfirmOrderPageComponent> orders { get; set; }
+        private static string _pageUrl = "/profile/current_orders/";
 
         public CurrentOrdersPage(IWebDriver driver) : base(driver)
         {
             PersonalInfoPageComponent = new PersonalInfoPageComponent(driver);
             NavigationMenuPageComponent = new NavigationMenuPageComponent(driver);
+            UserButton = new UserMenuHeaderButtonPageComponent(driver);
             FillOdersList();
         }
 
 
         private IWebElement _allButton => driver.FindElement(By.XPath("//span[contains(text(),'All')]/parent::span/parent::span/parent::a"));
         private IWebElement _waitingForConfirmButton => driver.FindElement(By.XPath("//span[contains(text(),'Waiting for confirm')]/parent::span/parent::span/parent::a"));
 
-        private int СountOrders()
+        public int СountOrders()
```

Comment:
It seems that there is cyrillic `C`. Please change to latin.

Reply:
> It seems that there is cyrillic `C`. Please change to latin.

This project builds even with the current name,  no matter Latin or Cyrillic. So you should not request the changes, but only comment, because this stops merge.